third_party/helm/helm.go

```diff
@@ -194,6 +194,13 @@ func (m *Manager) RunInstall(opts ...Option) error {
 	return m.run(o)
 }
 
+// RunUninstall provides a way to uninstall the specified helm chart (useful in teardowns etc...)
+func (m *Manager) RunUninstall(opts ...Option) error {
```

Comment:
It's been useful to us mostly when we decide that we don't want to create a Kind cluster but use an existing one (which could also be a non-Kind cluster). In that case I think it becomes essential so that we can properly clean up after the test.

Any thoughts @vladimirvivien and @cpanato?